Cargo.lock

```diff
@@ -265,6 +265,23 @@ dependencies = [
  "zbus",
 ]
 
+[[package]]
+name = "ashpd"
+version = "0.11.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "6cbdf310d77fd3aaee6ea2093db7011dc2d35d2eb3481e5607f1f8d942ed99df"
+dependencies = [
+ "enumflags2",
+ "futures-channel",
+ "futures-util",
+ "rand 0.9.0",
+ "serde",
+ "serde_repr",
+ "tokio",
+ "url",
+ "zbus",
+]
+
```

Comment:
That's still not nice. https://github.com/PolyMeilex/rfd/pull/240 is now merged, just not yet released.